### PR TITLE
In binary artifacts, nest includes deeper

### PR DESCRIPTION
**Pull request by almarklein**
Follow-up on #417

@Sahnvour, should the lib-dir also be nested?

### CODE REVIEW

**Comment by Sahnvour** [comment]
I don't think so, lgtm !

**Comment by almarklein** [comment]
image

File: Makefile [hunk 1]
```diff
@@ -53,12 +53,11 @@ package: lib-native lib-native-release
...
-		mkdir dist/$$ARCHIVEDIR; \
-		mkdir dist/$$ARCHIVEDIR/include; \
-		mkdir dist/$$ARCHIVEDIR/lib; \
+		mkdir -p dist/$$ARCHIVEDIR/include/webgpu; \
+		mkdir -p dist/$$ARCHIVEDIR/lib; \
 		cp ./dist/wgpu-native-git-tag                 dist/$$ARCHIVEDIR; \
-		cp ./ffi/webgpu-headers/webgpu.h              dist/$$ARCHIVEDIR/include; \
-		cp ./ffi/wgpu.h                               dist/$$ARCHIVEDIR/include; \
+		cp ./ffi/webgpu-headers/webgpu.h              dist/$$ARCHIVEDIR/include/webgpu; \
+		cp ./ffi/wgpu.h                               dist/$$ARCHIVEDIR/include/webgpu; \
```
Comment (rajveermalviya) [inline Makefile]: Does `wgpu.h` need to be under `include/webgpu` too?

Comment (rajveermalviya) [reply Makefile]: I feel this should be `include/wgpu.h` or `include/wgpu/wgpu.h` or `include/wgpu-native/wgpu.h`.
I'd prefer the ~~first one~~ first two.

Comment (almarklein) [reply Makefile]: I really don't know what would be best practice here. @Sahnvour what do you think?

Comment (Sahnvour) [reply Makefile]: I think all of the options are fine, afaik it's not an issue for this header since you own it and can dictate its standard location.

**Comment by almarklein** [comment]
Done:
image

**Review by rajveermalviya — APPROVED** [review]

**Comment by almarklein** [comment]
What's up with clippy on Aindroid builds?

**Comment by rajveermalviya** [comment]
This'll need a rebase.

**Pull request merged**